components/03-modules/form/form--compact.hbs

```diff
@@ -0,0 +1,9 @@
+<form class="form-compact {{ class }}">
```

Comment:
But it's a variant (additional template form--compact.hbs). Default one is without classes and you can add some custom.